config.go

```diff
@@ -58,26 +58,23 @@ type PumpConfig struct {
 	// Its default value is `0` seconds, which means that the pump will wait for the writing
 	// operation forever.
 	//
-	// An example of this configuration would be:
+	// The format for this configuration is:
 	// ```{.json}
-	// "mongo": {
-	//   "type": "mongo",
+	// "pump_name": {
+	//   ...
 	//   "timeout":5,
-	//   "meta": {
-	//     "collection_name": "tyk_analytics",
-	//     "mongo_url": "mongodb://username:password@{hostname:port},{hostname:port}/{db_name}"
-	//   }
+	//   "meta": {...}
 	// }
 	// ```
 	//
 	// In case that any pump doesn't have a configured timeout, and it takes more seconds to write
 	// than the value configured for the purge loop in the `purge_delay` config option, you will
-	// see the following warning message: `Pump PMP_NAME is taking more time than the value
+	// see the following warning message: `Pump {pump_name} is taking more time than the value
 	// configured of purge_delay. You should try to set a timeout for this pump.`.
```

Comment:
Perhaps replace this paragraph with:

Tyk will inform you if the pump's write operation is taking longer than the purging loop (configured via `purge_delay`) as this will mean that data is purged before being written to the target data sink.

If there is no `timeout` configured, the following warning log will be generated if this occurs:
`Pump {pump_name} is taking more time than the value configured of purge_delay. You should try to set a timeout for this pump.`